### DISCUSSION TITLE
Files in bin/mechanics

### DISCUSSION THREAD

**Discussion by ezrahaddad** [sritchie/sicm#10] [General]
Hi again,

Apologies for the delay, there's a half a day time difference between us I think. So, I followed the instructions in the read-me and got to the point where I can run the exercises in Quartz :D i'm still troubleshooting with spacemacs... 

I wasn't sure why but when i tried to load some of the exercises in bin/mechanics from the other chapters I couldn't(in quartz). I just got a message saying the file couldn't be found. This was the case for every chapter apart from 1 and 9. I saw that on your website you're doing a cool write up for each exercise but I wasn't sure if this was in pace with the exercises in bin/mechanics or if there were more that I just couldn't find.

Many thanks

**Comment by sritchie** [comment]
Hey @ezrahaddad , I think what you've discovered is that I haven't completed anything outside of those two chapters! I ended up making templates for everything so I'd know what I had left, but I've been sidetracked for quite a while on the Clojure port of of the "scmutils" library: https://github.com/sicmutils/sicmutils/

That can currently handle everything in both books, but I don't quite have my exercises ported over yet.

Just to be clear, it sounds like chapters 1 and 9 are doing the trick for you? Did you find the [markdown export](https://github.com/sicmutils/sicm-exercises/blob/master/md/1_lagrangian_mechanics.md) of chapter 1 too? I can't believe I haven't added a link for it yet, but if you install [this chrome plugin](https://chrome.google.com/webstore/detail/mathjax-3-plugin-for-gith/peoghobgdhejhcmgoppjpjcidngdfkod?hl=en) all of the math on that page will show up nicely too.

If you want to chat live, come say hi in the [#sicmutils](https://clojurians.slack.com/archives/C01ECA9AA74) channel on the Clojurians slack.

Thanks for working with the exercises here and sending me this report! Let me know if this helps and let's talk more.
